### PR TITLE
Refactor uts of adminserver

### PR DESCRIPTION
**Pull request by ywk253100**

### CODE REVIEW

File: src/adminserver/systemcfg/store/encrypt/driver_test.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+func (f *fakeCfgStore) Write(cfgs map[string]interface{}) error {
+	f.cfgs = cfgs
+	return nil
+}
+
+type fakeEncryptor struct {
+}
+
+func (f *fakeEncryptor) Encrypt(plaintext string) (string, error) {
```
Comment (reasonerjt) [inline src/adminserver/systemcfg/store/encrypt/driver_test.go]: The f.Decrypt(f.Encrypt(txt)) != txt

File: src/adminserver/systemcfg/encrypt/encrypt.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+// Encryptor encrypts or decrypts a strings
+type Encryptor interface {
+	// Encrypt encrypts plaintext
+	Encrypt(string) (string, error)
+	// Decrypt decrypts ciphertext
+	Decrypt(string) (string, error)
+}
+
+// AESEncryptor uses AES to encrypt or decrypt string
+type AESEncryptor struct {
```
Comment (reasonerjt) [inline src/adminserver/systemcfg/encrypt/encrypt.go]: The naming is a bit misleading, imagine we shift to other method to encrypt/decrypt.

File: src/adminserver/api/cfg_test.go [hunk 3]
```diff
@@ -18,196 +18,141 @@ package api
...
+	return f.err
+}
 
-	r, err := http.NewRequest("GET", "", nil)
-	if err != nil {
-		t.Errorf("failed to create request: %v", err)
-		return
+func TestListCfgs(t *testing.T) {
+	// 500
+	systemcfg.CfgStore = &fakeCfgStore{
```
Comment (reasonerjt) [inline src/adminserver/api/cfg_test.go]: love it!!!

File: src/adminserver/systemcfg/store/encrypt/driver_test.go (hunk 1 above)
Comment (ywk253100) [inline src/adminserver/systemcfg/store/encrypt/driver_test.go]: It doesn't matter. We just make sure the Encrypt() or Decrypt() is called by callers. We are not testing the functions of the fakeEncryptor.

**Pull request merged**